extensions/cornerstone/src/services/ViewportService/CornerstoneViewportService.ts

```diff
@@ -526,27 +529,57 @@ class CornerstoneViewportService extends PubSubService implements IViewportServi
    *          was initiated.
    * @return the viewportId that the measurement should be displayed in.
    */
-  public getViewportIdToJump(
-    activeViewportId: string,
-    displaySetInstanceUID: string,
-    cameraProps: unknown
-  ): string {
-    const viewportInfo = this.getViewportInfo(activeViewportId);
-
-    if (viewportInfo.getViewportType() === csEnums.ViewportType.VOLUME_3D) {
-      return null;
+  public getViewportIdToJump(activeViewportId: string, metadata): string {
```

Comment:
Could you please add extensive documentation to the function about the order in which we jump? It seems there are multiple orders and priorities that could benefit from a detailed explanation.